<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>DateNavigationWidget</class>
 <widget class="QMainWindow" name="DateNavigationWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>397</width>
    <height>373</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Navigation window by date</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <property name="spacing">
     <number>6</number>
    </property>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <property name="spacing">
       <number>0</number>
      </property>
      <item>
       <spacer name="horizontalSpacer">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeType">
         <enum>QSizePolicy::Expanding</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>5</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <widget class="QToolButton" name="tButtonExpandAll">
        <property name="toolTip">
         <string>Expand All</string>
        </property>
        <property name="statusTip">
         <string>Expand All</string>
        </property>
        <property name="whatsThis">
         <string>Expand All</string>
        </property>
        <property name="text">
         <string>Expand All</string>
        </property>
        <property name="icon">
         <iconset resource="../NavigationPanel.qrc">
          <normaloff>:/expand all</normaloff>:/expand all</iconset>
        </property>
        <property name="autoRaise">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QToolButton" name="tButtonCollapseAll">
        <property name="toolTip">
         <string>Collapse All</string>
        </property>
        <property name="statusTip">
         <string>Collapse All</string>
        </property>
        <property name="whatsThis">
         <string>Collapse All</string>
        </property>
        <property name="text">
         <string>Collapse All</string>
        </property>
        <property name="icon">
         <iconset resource="../NavigationPanel.qrc">
          <normaloff>:/collapse all</normaloff>:/collapse all</iconset>
        </property>
        <property name="autoRaise">
         <bool>true</bool>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <widget class="QGroupBox" name="groupBox">
      <property name="title">
       <string>Traced date</string>
      </property>
      <layout class="QGridLayout" name="gridLayout">
       <item row="0" column="0">
        <widget class="QRadioButton" name="rButtonCreationDate">
         <property name="toolTip">
          <string>Creation date</string>
         </property>
         <property name="statusTip">
          <string>Creation date</string>
         </property>
         <property name="whatsThis">
          <string>Creation date</string>
         </property>
         <property name="text">
          <string>Creation date</string>
         </property>
         <property name="checked">
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item row="1" column="0">
        <widget class="QRadioButton" name="rButtonModificationDate">
         <property name="toolTip">
          <string>Modification date</string>
         </property>
         <property name="statusTip">
          <string>Modification date</string>
         </property>
         <property name="whatsThis">
          <string>Modification date</string>
         </property>
         <property name="text">
          <string>Modification date</string>
         </property>
         <property name="checked">
          <bool>false</bool>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
    <item>
     <widget class="QTreeView" name="treeView">
      <property name="headerHidden">
       <bool>true</bool>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources>
  <include location="../NavigationPanel.qrc"/>
 </resources>
 <connections/>
</ui>
